test/v1_0_2/non_templating.js

```diff
@@ -1038,7 +1017,7 @@
                     } else {
                         var result = parse(res.body, done);
                         expect(result).to.have.property('more');
-                        Joi.assert(result.more, Joi.string().uri());
+                        Joi.assert(result.more, Joi.string().regex(/(\/[\w\.\-]+)+\/?/));
```

Comment:
More IRLs are relative. Better regex to use?
https://github.com/adlnet/xAPI-Spec/blob/master/xAPI.md#42-retrieval-of-statements